merlin-server/src/test/java/gov/nasa/jpl/aerie/merlin/server/services/TypescriptCodeGenerationServiceTest.java

```diff
@@ -15,7 +15,7 @@ class TypescriptCodeGenerationServiceTest {
   @Test
   void testCodeGen() throws MissionModelService.NoSuchMissionModelException, NoSuchPlanException {
     final var codeGenService = new TypescriptCodeGenerationServiceAdapter(new StubMissionModelService(), new StubPlanService());
-
+    final var expected = codeGenService.generateTypescriptTypes("abc", Optional.of(new PlanId(1L)));
     assertEquals(
```

Comment:
Minor: `expected` should be the first parameter in this `assertEquals`.

Reply:
ok